doc/en/invocation-fixture.rst

```diff
@@ -0,0 +1,61 @@
+.. _invocation_scoped_fixture:
+
+Invocation-scoped fixtures
+==========================
+
+.. versionadded:: 3.0
+
+.. note::
+    This feature is experimental, so if decided that it brings too much problems
+    or considered too complicated it might be removed in pytest ``3.1``.
+
+    ``tmpdir`` and ``monkeypatch`` might become ``invocation`` scoped
```

Comment:
i suggest to already make monkeypatch  an invocation scoped fixture.  Maybe even tmpdir?   This way we are going to hear very quickly if there are problems :)